brainscore/model_interface.py

```diff
@@ -12,7 +12,14 @@ class BrainModel:
     """
 
     RecordingTarget = Enum('RecordingTarget', " ".join(['V1', 'V2', 'V4', 'IT']))
-    Task = Enum('Task', " ".join(['passive', 'probabilities', 'label', 'visual_search_obj_arr']))
+    """
+    location to record from
+    """
+
+    Task = Enum('Task', " ".join(['passive', 'probabilities', 'label', 'visual_search_obj_arr', 'visual_search']))
```

Comment:
what are the differences between `visual_search_obj_arr` and `visual_search`? Can we document this somewhere?

Reply:
again this is outdated. i renamed: 'object_search', 'visual_search'

recall the discussion with Mengmi. we have two benchmarks one benchmark for the experiment similar to the object array dataset. while the other one for waldo and natural design. 

object_search >> for object array
visual_search >> for other

Sure I will add the comments.